Source/WebCore/platform/graphics/avfoundation/CDMFairPlayStreaming.cpp

```diff
@@ -269,7 +269,7 @@ Vector<AtomString> CDMPrivateFairPlayStreaming::supportedInitDataTypes() const
 
 bool CDMPrivateFairPlayStreaming::supportsConfiguration(const CDMKeySystemConfiguration& configuration) const
 {
-    if (!WTF::anyOf(configuration.initDataTypes, [] (auto& initDataType) { return validInitDataTypes().contains(initDataType); })) {
+    if (!WTF::anyOf(configuration.initDataTypes, [] (auto& initDataType) { return validInitDataTypes().contains(AtomString { initDataType }); })) {
```

Comment:
Seems like we are using the wrong data type if we need to search `validInitDataTypes()` for strings that don’t happen to already be atom strings. Doing a second hash table lookup to turn the data type into an `AtomString` seems like a waste if we’re just going to look up in a second hash table.

Reply:
There is one other call site and it has an AtomString. Here the value comes from IDL so I can likely just get an AtomString right away from the JS bindings.